docs/code-examples/arrow3d_simple.rs

```diff
@@ -28,3 +32,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     rerun::native_viewer::show(storage.take())?;
     Ok(())
 }
+
+fn rnd(v: f64) -> f64 {
```

Comment:
The need for this detracts some from the example. If it's just here so we can get determinism on the round-trip-tests we should at least add a comment so we don't lead users down the path of thinking all angles and lengths need to be rounded for some reason.